Bookish/Views/Catalog/BookList.cshtml

```diff
@@ -7,17 +7,31 @@
 
 <!DOCTYPE html>
 
-<html>
-<head>
-    <table style="">
-         <tr>
-           <th>Title</th>
+<html lang="en">
+    <head>
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+        <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+        <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+
+        <title>Book List</title>
+    </head>
+<body>
+<div>
+    <table class="display" style="width:100%" id="myTable">

```

Comment:
Wouldn't use inline styles - look into using the classes available in Bootstrap, docs are [here](https://getbootstrap.com/docs/5.1/getting-started/introduction/)